agent/Dockerfile

```diff
@@ -20,4 +20,10 @@ ENV LOG_FILE_PATH="/var/log/agent.log" ANODOT_API_URL="http://api.anodot.com" EN
 ENV UWSGI_INI="/usr/src/app/uwsgi.ini"
 
 RUN pip install --upgrade pip && pip install -r requirements.txt && python setup.py install && echo ". /usr/src/app/agent-complete.sh" >> /root/.bashrc
-RUN apt update && apt install -y vim netcat curl nano
+
```

Comment:
Let's create a shell script, too many RUN commands and it's duplicated for the two images